staging/velero/values.yaml

```diff
@@ -192,11 +192,6 @@ configMaps: {}
 # Deploy a default Minio cluster to use as the backend for Velero.
 minioBackend: false
```

Comment:
Should not it be true?

Reply:
When we first put this chart together we didn't want to provide minio as enabled by default because it's intra-cluster and has worse characteristics for failure cases than say, using AWS S3. The addons that wrap this chart for Konvoy deployments however _do_ enable this by default.